cmd/newrelic/main.go

```diff
@@ -50,6 +52,8 @@ func init() {
 	Command.AddCommand(apiaccess.Command)
 
 	CheckPrereleaseMode(Command)
+
+	os.Setenv("CLI_VERSION", version)
```

Comment:
should we use the consistent NEW_RELIC_ prefix syntax?

Reply:
Good call, I'll update.